canonical-web-and-design/ubuntu.com
# Add wsl contact us

## Done
Check the commit messages

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/wsl
- Check that the [highlighted sections of the copy doc](https://docs.google.com/document/d/1M7e2B08GgPA7K6Wx0ThE6s9_xQINM0MSkarfdKUTEYI/edit#) have been applied.
- Click the "Contact us" button and check the interactive contact form matches [the copy doc](https://docs.google.com/document/d/1UxS6S6RTRIiI2rD-yOc8bfbVb2FP1Czd8mGejKrjoDE/edit#heading=h.dxn10eelvkoc)
- Fill in the form and check that works. _Remember the thank you page will load on live so you need to replace the live site domain with local or the demo domain_
- Go to /wsl/contact-us and see that works

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/8830
